certified-connectors/Docurain/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Docurain",
+    "description": "DocurainAPI Communication Connector. Docurain API allows you to create your own documents using template files registered in Docurain and data from PowerAppsn.",
```

Comment:
@geckoR42 `DocurainAPI Communication Connector. ` is not required in the description. Can you please let us know what is `PowerAppsn`. If it is a typo and "PowerApps", It is not allowed to use our product names in the description or title of the connector.

Reply:
@sriyen-msft Removed `DocurainAPI Communication Connector`.
"PowerApps" is a typo. Fixed.